index.html

```diff
@@ -959,19 +959,27 @@ <h2><dfn>Coalesced events</dfn></h2>
                 <figcaption>Example of a curve in a drawing application – using only the coalesced
                 coordinates from <code>pointermove</code> events (the grey dots), the curve is noticeably
                 angular and jagged; the same line drawn using the more granular points provided by
-                <code>getCoalescedEvents</code> (the red circles) results in a smoother approximation
+                <code>getCoalescedEvents()</code> (the red circles) results in a smoother approximation
                 of the pointer movement.</figcaption>
             </figure>
 
             <p>A <a>PointerEvent</a> has an associated <dfn>coalesced events list</dfn> (a list of
-            zero or more <code>PointerEvent</code>s). If a trusted event is a <code>pointermove</code> or
-            <code>pointerrawupdate</code> event, the list is a sequence of all <code>PointerEvent</code>s
+            zero or more <code>PointerEvent</code>s). For trusted <code>pointermove</code> and
+            <code>pointerrawupdate</code> events, the list is a sequence of all <code>PointerEvent</code>s
             that were coalesced into this event; otherwise it is an empty list.
```

Comment:
Nit (also in Sec10.2): without the old "if", the meaning of "otherwise" becomes less obvious.  I guess better to exclude untrusted events clearly, like:
> For all other trusted event types, it is an empty list.

Reply:
Ah yes, clearer. done